Zend/zend.c

```diff
@@ -2,12 +2,12 @@
    +----------------------------------------------------------------------+
    | Zend Engine                                                          |
    +----------------------------------------------------------------------+
-   | Copyright (c) Zend Technologies Ltd. (http://www.zend.com)           |
+   | Copyright (c) Zend Technologies USA, Inc. (https://www.zend.com)     |
    +----------------------------------------------------------------------+
-   | This source file is subject to version 2.00 of the Zend license,     |
+   | This source file is subject to version 2.01 of the Zend license,     |
    | that is bundled with this package in the file LICENSE, and is        |
    | available through the world-wide-web at the following url:           |
-   | http://www.zend.com/license/2_00.txt.                                |
+   | https://www.zend.com/sites/zend/files/pdfs/zend-engine-license-2.01.txt |
```

Comment:
I think the "nice" boxes need to be extended.

Reply:
I was trying to keep the patch minimal. Would like to hear from the release managers on this; @ramsey - what are your opinions?

Reply:
Or maybe we could remove `/pdfs/` from the URL given that the served file is not a PDF.

Could Zend devops move this file to `https://www.zend.com/sites/zend/files/zend-engine-license-2.01.txt` instead? Thanks!

Reply:
Unfortunately, no. :-/

Reply:
Could you rename the file to `zend-license-2.01.txt`?